blockmanager.go

```diff
@@ -221,6 +241,22 @@ func (b *blockManager) resetHeaderState(newestHash *wire.ShaHash, newestHeight i
 	}
 }
 
+// RelayTxDisabled returns whether or not relaying of transactions for the given
+// peer is disabled.
+// It is safe for concurrent access.
+func (b *blockManager) RelayTxDisabled(p *peer.Peer) bool {
```

Comment:
As mentioned in IRC, this will be changing to take a `*peerInfo` directly and the `peerInfo` struct will have a pointer to the peer.  I won't comment on all of the other cases, but just putting this here for reference.